roles/set_scale/defaults/main/config.yml

```diff
@@ -0,0 +1,9 @@
+---
+# The instance type to user for the machineset being scaled
+machineset_instance_type: "g4dn.xlarge"
```

Comment:
I think you can remove the default value here, as the instance type is mandatory in the toolbox script